cvat-ui/src/stylesheet.css

```diff
@@ -101,53 +82,28 @@
 }
 
 
-.ant-menu.cvat-header-menu {
-    width: -webkit-fit-content;
-    width: -moz-fit-content;
-    width: fit-content;
+.ant-dropdown-trigger.cvat-header-menu-dropdown {
+    display: -webkit-box;
```

Comment:
Don't you want to try [autoprefixer](https://github.com/postcss/autoprefixer) for adding prefixes?

Reply:
I am going to change css > less or sass soon, and use autoprefixer also